test/checks/shared/heading-whitespace-only.js

```diff
@@ -0,0 +1,192 @@
+describe('heading-whitespace-only', function() {
+	'use strict';
```

Comment:
some more edge cases:
- what happens when heading is given accessible name via `aria-*`? and they end up being space characters?
- what happens if heading element has shadowDOM that then exposes a non space character as heading content?

Reply:
I'll have to figure out how to handle those. Thanks for bringing them up.

Reply:
Definitely need to do some screen reader testing on the aria-label combinations